docbook packages no longer work properly

Trying to rebuild git after the lastest upload of docbook-{xml,xsl,xsl-ns}:
```
 xmlto: /C/_/git/src/git-2.35.1/Documentation/git-add.xml does not validate (status 3)
  xmlto: Fix document syntax or use --skip-validation option
  I/O error : Attempt to load network entity http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd
  /C/_/git/src/git-2.35.1/Documentation/git-add.xml:2: warning: failed to load external entity "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
  D DocBook XML V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
                                                                                 ^
  I/O error : Attempt to load network entity http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd
  warning: failed to load external entity "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
  validity error : Could not load the external subset "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd"
  Document /C/_/git/src/git-2.35.1/Documentation/git-add.xml does not validate
  make[1]: *** [Makefile:373: git-add.1] Error 13
```
https://github.com/jeremyd2019/MSYS2-packages/runs/4999000825?check_suite_focus=true

--- a/docbook-xml/PKGBUILD
+++ b/docbook-xml/PKGBUILD
@@ -2,7 +2,7 @@
 
 pkgname=docbook-xml
 pkgver=4.5
-pkgrel=3
+pkgrel=4
 pkgdesc="A widely used XML scheme for writing documentation and help"
 arch=('any')
 url="https://www.oasis-open.org/docbook/"

--- a/docbook-xml/docbook-xml.install
+++ b/docbook-xml/docbook-xml.install
@@ -13,11 +13,11 @@ post_install() {
     "file:///etc/xml/docbook-xml" \
     etc/xml/catalog
   usr/bin/xmlcatalog --noout --add "delegateSystem" \
-    "https://www.oasis-open.org/docbook/" \
+    "http://www.oasis-open.org/docbook/" \
     "file:///etc/xml/docbook-xml" \
     etc/xml/catalog
   usr/bin/xmlcatalog --noout --add "delegateURI" \
-    "https://www.oasis-open.org/docbook/" \
+    "http://www.oasis-open.org/docbook/" \
     "file:///etc/xml/docbook-xml" \
     etc/xml/catalog
 }

